script/update_beats.sh

```diff
@@ -7,6 +7,7 @@ BEATS_VERSION="${BEATS_VERSION:-master}"
 # Find basedir and change to it
 DIRNAME=$(dirname "$0")
 BASEDIR=${DIRNAME}/../_beats
+rm -rf $BASEDIR
```

Comment:
doesn't this prevent us from owning any files in `_beats/` ourselves?  Maybe that's the goal, but we historically have overridden a few files, including `dev-tools/packer/readme.md.j2` which doesn't actually exist any more (will check what's in our packages these days next).

Reply:
following up on that readme, all is well - https://github.com/elastic/apm-server/blob/master/package-README.md.tmpl takes care of our customizations now.  Let's just remove that exclude since it does nothing anyway, looks like it should be fine to not own anything under `_beats`

Reply:
As we don't have any files that we overwrite, I thought it was a good idea to clean out the `_beats` folder regularly, as otherwise we keep files that get removed in beats, which can be confusing, e.g. the [fields.generated.yml](https://github.com/elastic/apm-server/pull/1272/files#diff-66cab9e9cbfa9dc1c2c954f7b0d5f880) that doesn't exist anymore.